src/components/input-field/input-field.tsx

```diff
@@ -723,6 +724,14 @@ export class InputField {
         }
     }
 
+    private getType() {
+        if (this.type === 'url') {
+            return 'text';
+        }
+
+        return this.type;
+    }
+
```

Comment:
I'm not sure this is the correct place for this bugfix. This will more or less remove the feature of having URL fields altogether, so it will no longer work for consumers using this field outside of a form either.

This should probably be fixed on a schema level since it is in the schema that we have defined that we should use a URL field. However, I think it is a missed opportunity to not use validation for stuff like this. If a URL is required here we should indicate if the user has entered something that is not valid. Sure, it might be annoying at the moment but we are not using the validation status of the form to determine if a user can save or not. The field will also not display an error until it is interacted with in some way.

Maybe you discussed all this in the project before I joined. What are your thoughts? :slightly_smiling_face:

Reply:
Isn't the only difference between the types, url and text, that url is validated? 

I think it is nice to only have to write the url without specifying the protocol. I thought that it should always be the same for the user if they should specify the full url or not. Otherwise, I think it might be confusing if they sometimes only need to write a part of the url and in another part of the application must write the entire part of the url. That is why I put it in lime-elements.  So I think if we don't want to fix it in lime-element we shouldn't fix it at all, so it is consistent thought the application. 🙂

Reply:
I don't think we discussed it really, we just compared it with the old card design and then said it was a bug 😅

Reply:
> Isn't the only difference between the types, url and text, that url is validated?

Yes, but with this change we remove that possibility altogether so this will be a breaking change :slightly_smiling_face: Also, `type='url'` will add an icon that you can click to navigate to that URL.

> I think it is nice to only have to write the url without specifying the protocol.

But then it's not a URL :smile:

> I thought that it should always be the same for the user if they should specify the full url or not. Otherwise, I think it might be confusing if they sometimes only need to write a part of the url and in another part of the application must write the entire part of the url. That is why I put it in lime-elements. So I think if we don't want to fix it in lime-element we shouldn't fix it at all, so it is consistent thought the application. :slightly_smiling_face:

Yes that is true, but it might also depend in what context the user is in when doing that :thinking: 
Either way, I'm not sure how much you have already discussed around this issue, but if this should be fixed I do not think we should put it on this level. Lime Elements can be used by other parties as well, and those might want the URL `type` even if we decide not to use it in the webclient :slightly_smiling_face:

Reply:
> I don't think we discussed it really, we just compared it with the old card design and then said it was a bug

Haha ok :sweat_smile:

Reply:
Okey, I can move it then or should we wait with this one until the issue is discussed. I don't know what I should do. If I shouldn't fix it at all or if I should move it? 🤔

Reply:
The problem if we don't fix it is that, if a user has an invalid url (that worked before the new card), it will not work to click on the trailing icon to go to that page.

Reply:
Yeah ok, but that we could probably fix though so that invalid URLs are still clickable
We cannot know the correct URL but we could still guess and just prefix it with either `http` or `https`

Reply:
I'm going to merge this without that fix, for now, since there are still improvements here.